build.sbt

```diff
@@ -252,3 +230,173 @@ fixCSS := {
   if (("npm run fix-dark" #&& "npm run fix-light" !) != 0)
     throw new Exception("Error in CSS fix")
 }
+
+val pushCond                 = "github.event_name == 'push'"
+val prCond                   = "github.event_name == 'pull_request'"
+val masterCond               = "github.ref == 'refs/heads/master'"
+val geminiRepoCond           = "startsWith(github.repository, 'gemini')"
+val notDependabotCond        = "github.actor != 'dependabot[bot]'"
+def allConds(conds: String*) = conds.mkString("(", " && ", ")")
+def anyConds(conds: String*) = conds.mkString("(", " || ", ")")
+
+lazy val setupNode = WorkflowStep.Use(
+  UseRef.Public("actions", "setup-node", "v2"),
+  name = Some("Use Node.js"),
+  params = Map("node-version" -> "14", "cache" -> "npm"),
+  env = Map("FONTAWESOME_NPM_AUTH_TOKEN" -> "${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}")
```

Comment:
I guess we need this only when we do the final packaging

Reply:
It seems the cache step fails unless it has the `FONTAWESOME_NPM_AUTH_TOKEN` in the env.

Reply:
So the token gives us rights to use the paid version of the icons in font awesome. I think it is only required when we do dev but maybe it also fails when doing npm install, Would it do it in this case?

Reply:
Yes, if you want to use the node action with the npm cache step, it fails without the token. Probably because the cache step needs it.

Reply:
Yeah, we ran into that issue awhile back, IIRC.